api/src/main/java/io/minio/ErrorCode.java

```diff
@@ -54,7 +54,6 @@
   INVALID_BUCKET_NAME("InvalidBucketName", "Bucket name is not valid"),
   INVALID_BUCKET_STATE("InvalidBucketState", "The request is not valid with the current state of the bucket"),
   INVALID_DIGEST("InvalidDigest", "Specified Content-MD5 is not valid"),
-  INVALID_ENCRYPTION_ALGORITHM_ERROR("InvalidEncryptionAlgorithmError", "Invalid encryption algorithm error"),
```

Comment:
Capturing more server error code is better than removing them.  This can not be removed.

Reply:
restored.